src/main/deliferoo/view/MapView.java

```diff
@@ -140,40 +151,39 @@ public void drawMarker(Delivery delivery, Color color, Integer markerSize) {
     }
 
     /**
-     * Draws the paths followed by a round of deliveries
+     * Draws the paths followed by a round of deliveries, removes old round
      *
-     * @param bestPaths		a list of best paths to take to optimally complete the round
-     * @param color 		the color of the round drawn
-     * @param strokeWidth	the width of the round drawn
+     * @param bestPaths   a list of best paths to take to optimally complete the
+     *                    round
      */
-    public void drawRound(List<BestPath> bestPaths, Color color, Integer strokeWidth) {
-	Polyline round = new Polyline();
-
-	round.setStroke(color);
-	round.setStrokeWidth(strokeWidth);
-
+    public void updateRound(List<BestPath> bestPaths) {
+	this.mapView.getChildren().remove(this.round);
```

Comment:
Maybe separate into 2 methods? draw and update, but update will simply be remove and draw.